subgraphs/balancer-forks/src/modules/Rewards.ts

```diff
@@ -116,7 +114,13 @@ export function updateFactoryRewards(
   );
   pool.save();
   //////////////////////////////////////////////////////////////////
+}
 
+export function updateFactoryRewards(
+  poolAddress: Address,
+  gaugeAddress: Address,
+  block: ethereum.Block
+): void {
```

Comment:
Won't this miss the stakedOutTokenAmount of the side chains though? Are you thinking there is not a way to calculate this for chains that are not Ethereum mainnet? My impression was that the updateControllerRewards() will only end up updating the stakedOutputTokenAmount for Ethereum mainnet if it is left there, because this code will only get executed on Ethereum mainnet.